### DISCUSSION TITLE
Downgrade Pic Firmware

### DISCUSSION THREAD

**Discussion by PvdGulik** [rvdbreemen/OTGW-firmware#50] [Q&A]
I would like to downgrade the PIC-firmware to 4.3. I tried both symbols behind gateway-4.3.hex, but nothing happens, and device info keep telling firmware is 5.1. Am I doing something wrong?

**Comment by rvdbreemen** [comment]
Well, I have been able to downgrade and upgrade with my setup. 

Not sure what is wrong. You could try The nodemcu by Schelte Bron. See if that works for you. 

Please let me know if that works for you. 

Link: https://otgw.tclcode.com/download.html#otgwmcu
